tests/integration/test_security.py

```diff
@@ -51,14 +45,14 @@ def test_cors_s3_override(self):
                 }
             ]
         }
-        bucket_name = "my-s3-bucket"
+        bucket_name = f"bucket-{short_uid()}"
 
         try:
-            client.create_bucket(Bucket=bucket_name)
-            client.put_bucket_cors(Bucket=bucket_name, CORSConfiguration=BUCKET_CORS_CONFIG)
+            aws_stack.get_or_create_bucket(bucket_name)
+            s3_client.put_bucket_cors(Bucket=bucket_name, CORSConfiguration=BUCKET_CORS_CONFIG)
```

Comment:
while we're refactoring this test to pytest, we could maybe also simplify the test by injecting the `s3_bucket` fixture, which gives you the name of a newly created s3 bucket, that is also recursively cleared and deleted after the test run (so we can remove the try/finally block)

Reply:
Good catch, thanks! 👍 Updated..